frontend/app/src/components/Sidebar/Sidebar.tsx

```diff
@@ -363,4 +363,5 @@ function SidebarWithProvider(props: SidebarProps): ReactElement {
   )
 }
 
-export default withTheme(SidebarWithProvider)
+const SidebarWithProviderAndTheme = withTheme(SidebarWithProvider)
```

Comment:
A bit unrelated, but I think the `withTheme` hook should be removed and replaced with `useTheme`. `withTheme` is an old relict from the class-based component world, but shouldn't be used for functional components.

Reply:
https://github.com/streamlit/streamlit/pull/9812 removes all the `withTheme` uses that can be removed and adds a linter rule. The only case that can be removed and not covered by the PR is Sidebar.

Reply:
I included the removal of `withTheme` in Sidebar in https://github.com/streamlit/streamlit/pull/9812